diff --git a/router/router-proxy.js b/router/router-proxy.js
index 1bf15779f7d0e0658e9d50c67c9924ffc7a70867..256e1eee331a878155bffe7c0ec8ebc335790f9c 100644
--- a/router/router-proxy.js
+++ b/router/router-proxy.js
@@ -13,6 +13,7 @@ class RouterProxy {
                 exceptionsHandler.next(e, host);
                 return res;
             }
+            next();
         };
     }
     createExceptionLayerProxy(targetCallback, exceptionsHandler) {
@@ -25,6 +26,7 @@ class RouterProxy {
                 exceptionsHandler.next(e, host);
                 return res;
             }
+            next();
         };
     }
 }
